backend/src/embedding/openai-embbeding-provider.ts

```
@@ -0,0 +1,33 @@
+  }
+
+  static async getEmbList() {
+    try {
+      const models = await this.openai().models.list();
+      Object.values(models).filter((model) => model.object === 'embedding');
+      this.logger.log(`Models fetched: ${models.data.length}`);
+    } catch (error) {
+      this.logger.error('Error fetching models:', error);
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)